scripts/verify-replace.sh

```diff
@@ -16,27 +19,45 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    ERRFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-error.XXX)
     echo "Cloning repo in /tmp"
     git clone --depth=1 ${repo} ${repo_path}
     echo "Repo cloned successfully"
     cd ${repo_path}
     if ! make pre-verify; then
-        ERRORLIST+="($(basename ${repo}))"
+        ERRORREPOLIST+="$(basename ${repo}) "
         continue
     fi
     echo "Initiating 'go mod replace' of current toolchain common version in dependent repos"
     go mod edit -replace github.com/codeready-toolchain/toolchain-common=${C_PATH}
-    make verify-dependencies || ERRORLIST+="($(basename ${repo}))"
+    make verify-dependencies &> ${ERRFILE} 
+    rc=$?
+    if [ ${rc} -ne 0 ]; then
+    ERRORREPOLIST+="$(basename ${repo}) " 
+    ERRORFILELIST+="${ERRFILE}  "
+    fi
     echo                                                          
     echo =========================================================================================
     echo                                                           
 done
-if [ ${#ERRORLIST[@]} -ne 0 ]; then
+echo                "Summary"
+if [ ${#ERRORREPOLIST[@]} -ne 0 ]; then
     echo "Below are the repos with error: "
-    for e in ${ERRORLIST[*]}
+    for e in ${ERRORREPOLIST[*]}
     do
         echo "${e}"
     done
+    for c in ${ERRORFILELIST[*]}
+    do  
+        echo                                                          
+        echo =========================================================================================
+        echo 
+        echo                       "${c} has the following errors "
+        echo                                                          
+        echo =========================================================================================
+        echo
+        cat "${c}" | grep "${GOLINTREGEX}\|${ERRORREGEX}"
```

Comment:
since you use regexps, shouldn't it has `grep -E`?

Reply:
I used quotes instead of `-E`, which did the job.
but i have now used `-E` in the command